### PR TITLE
feat(alarms): add 5xx alarm

### PR DESCRIPTION
**Pull request by jpetto**
## Goal

add relevant alarms to the service.

please check the validity of the alarms configured, specifically the `httpLatency` alarm. do the parameters make sense?

## Implementation Decisions

not configuring an `HTTPRequestCount` alarm as we don't have an idea of request volume at this time. we could take a guess and make a non-critical alarm though. open to thoughts here.

### CODE REVIEW

File: .aws/src/main.ts [hunk 1]
```diff
@@ -172,17 +172,26 @@ class FirefoxAndroidHomeRecommendations extends TerraformStack {
...
         httpLatency: {
-          evaluationPeriods: 2,
-          threshold: 500,
-          actions: config.isDev ? [] : [],
+          // making this threshold 700 because, on a cache miss, this service
+          // will need to hit client API, which has to hit Recs API, so it the
+          // response could be a little slow. results are cached for 30 minutes
+          threshold: 700,
+          evaluationPeriods: 1,
+          period: 300,
```
Comment (sri-kirsh) [inline .aws/src/main.ts:191]: is it ok to leave the `evaluationPeriods` as 1 for a critical alarm? usually in other repo, we have them atleast 2

Comment (jpetto) [reply .aws/src/main.ts:191]: hmm, good call. i agree a single response over 700ms is not worthy of being woken up. i could even see this being 3.

Comment (jpetto) [reply .aws/src/main.ts:193]: @sri-kirsh updated to 3 periods - what do you think?

**Comment by bassrock** [comment]
@jpetto per our conversations, shall we remove latency alarms?

**Review by bassrock — APPROVED** [review]

File: .aws/src/main.ts [hunk 2]
```diff
@@ -172,17 +172,14 @@ class FirefoxAndroidHomeRecommendations extends TerraformStack {
...
           evaluationPeriods: 2,
           period: 600,
-          actions: config.isDev ? [] : [],
-        },
-        httpLatency: {
-          evaluationPeriods: 2,
-          threshold: 500,
-          actions: config.isDev ? [] : [],
+          actions: config.isDev
+            ? [pagerDuty.snsNonCriticalAlarmTopic.arn]
```
Comment (bassrock) [inline .aws/src/main.ts:181]: @jpetto actually remove the alarm actions on dev

**Review by bassrock — APPROVED** [review]

**Pull request merged**